packages/bundled-angular-compiler/src/index.ts

```diff
@@ -1,5 +1,10 @@
 export * from '@angular/compiler';
 
-import type { MessageBundle, ParsedTemplate } from '@angular/compiler';
+import type {
+  MessageBundle,
+  ParsedTemplate,
+  LocalizedString,
+} from '@angular/compiler';
 export type Comment = Required<ParsedTemplate>['commentNodes'][number];
 export type Message = ReturnType<MessageBundle['getMessages']>[number];
+export type I18nMeta = LocalizedString['metaBlock'];
```

Comment:
I think this makes sense to expose this here like the other types, but can also just inline it into `packages/eslint-plugin-template/src/rules/i18n.ts`